### PR TITLE
add easyconfigs for USEARCH 5.2.236 and 6.1.544

### PR DESCRIPTION
**Pull request by boegel**
fleshed out from #1057

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/88e379510446fae04aff for a full test report.

**Comment by boegel** [comment]
@wpoely86: please review?

File: easybuild/easyconfigs/u/USEARCH/USEARCH-5.2.236-i86linux32.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+easyblock = 'Binary'
+
+name = 'USEARCH'
+version = '5.2.236'
+versionsuffix = '-i86linux32'
```
Comment (wpoely86) [inline easybuild/easyconfigs/u/USEARCH/USEARCH-5.2.236-i86linux32.eb]: why only the 32 bit version?

Comment (boegel) [inline easybuild/easyconfigs/u/USEARCH/USEARCH-5.2.236-i86linux32.eb]: because that's all I need :)

cfr the 32/64-bit combo easyconfig in #1057

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**